src/components/program/ProgramListComponent.tsx

```diff
@@ -1,45 +1,25 @@
 import React, { useState, useEffect } from 'react';
-import { ProgramModel } from '../../objectModel/ProgramModel';
+import ProgramModel from '../../objectModel/ProgramModel';
 import ProgramService from '../../services/ProgramService';
 
-const ProgramListComponent = () => {
+const ProgramListComponent = function () {
```

Comment:
Are we not doing arrow functions? I would prefer `() =>` to `function ()` personally

Reply:
airbnb lint style prefers function, otherwise gives error.  I added issue to evaluate whether to configure airbnb differently.